zipkin-web/src/main/scala/com/twitter/zipkin/web/Main.scala

```diff
@@ -53,7 +44,6 @@ trait ZipkinWebFactory { self: App =>
   )
 
   val webServerPort = flag("zipkin.web.port", new InetSocketAddress(8080), "Listening port for the zipkin web frontend")
-  val webRootUrl = flag("zipkin.web.rootUrl", "http://localhost:8080/", "Url where the service is located")
```

Comment:
removed this from docker as twitter server is very sensitive about being passed more properties than needed https://github.com/openzipkin/docker-zipkin/commit/d97ac3402e20bd4aca295fdead11b48d40416345